### DISCUSSION TITLE
Call prefix in phone app

### DISCUSSION THREAD

**Discussion by rudolfmu** [FossifyOrg/General-Discussion#154] [Ideas]
Could you please add a field where a general prefix string can be entered which would be inserted ahead of numbers called. This could contain codes for calling cards, USSD code for surpressing caller id, pbx codes and such.
Right now I must store duplicate entries, with and without prefix.

**Comment by Aga-C** [comment]
Please create a new issue via https://github.com/FossifyOrg/Phone/issues/new/choose and fill properly the form.
